apps/src/blockly/addons/cdoUtils.js

```diff
@@ -11,3 +11,7 @@ export function getBlockFields(block) {
   });
   return fields;
 }
+
+export function isWorkspaceReadOnly(workspace) {
+  return false; // TODO
```

Comment:
It might be worth keeping the `used for feedback` from the cdoWorkspaceSvg.js to maintain the context.